library/neo4j

```diff
@@ -1,17 +1,129 @@
-# maintainer: Ben Butler-Cole <ben@neotechnology.com> (@benbc)
+Maintainers: Ben Butler-Cole <ben@neotechnology.com> (@benbc),
+             Jonas Kalderstam <jonas.kalderstam@neotechnology.com> (@spacecowboy),
+             Steven R Baker <steven.baker@neotechnology.com> (@srbaker)
 
-2.3.4: git://github.com/neo4j/docker-neo4j@3f379d33f5d08f2e483769427ab1e0b898a819ac 2.3.4
-2.3.5: git://github.com/neo4j/docker-neo4j@3f379d33f5d08f2e483769427ab1e0b898a819ac 2.3.5
-2.3: git://github.com/neo4j/docker-neo4j@3f379d33f5d08f2e483769427ab1e0b898a819ac 2.3.5
+Comment: Old point releases are included in this library file so that
+         users are not forced to upgrade to the latest point releases
+         in order to get security fixes to the base image. See
+         https://github.com/docker-library/official-images/issues/1864
+         for a discussion of this point.
```

Comment:
@yosifkit Sure, I'll do that. I was trying to follow RFC 2822. If the format isn't exactly the same as that, maybe you could mention the differences in the README.